scripts/ci/images/ci_run_docker_tests.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import argparse
+import shlex
+import subprocess
+import sys
+from pathlib import Path
+from typing import List
+
+AIRFLOW_SOURCE = Path(__file__).resolve().parents[3]
```

Comment:
This `3` is (too) magical. What does it do?

Reply:
Just 3 levels up in directory tree.. I agree it's probably better than (abspath(join('..', .... but a comment might be useful

Reply:
I think my main problem is actually that is’s entirely unclear why it’s three levels up. Also either of these might be better

```python
Path(__file__).resolve().parent.parent.parent

# or

Path(__file__, "..", "..", "..").resolve()
```

Reply:
Agree. either of the two is better.